packages/graph/hash_graph/lib/graph/src/store/postgres/mod.rs

```diff
@@ -643,6 +644,30 @@ where
 
         Ok(entity_id)
     }
+
+    async fn update_link_active(
+        &self,
+        active: bool,
```

Comment:
What do we feel about bool parameters. Should we replace by an enum that is more descriptive?

Reply:
Addressed in fcf57780208a7877765ecc584d769ea09bc6fa49